_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -52243,26 +52076,47 @@
 /turf/open/floor/plasteel/white,
 /area/medical/medbay/central)
 "clm" = (
-/obj/item/radio/intercom{
-	pixel_y = -30
-	},
-/turf/open/floor/plasteel/white,
-/area/medical/medbay/central)
-"cln" = (
+/obj/item/twohanded/required/kirbyplants,
```

Comment:
This is a fixed type, use obj/item/twohanded/required/kirbyplants/random for some added soul